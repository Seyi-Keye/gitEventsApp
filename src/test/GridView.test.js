import React from "react";
import GridView from "../GridView";
import renderer from "react-test-renderer";

describe("GridView component", () => {
  let wrapper = renderer.create(<GridView />);
  it("renders succesfully", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  it("renders without props", () => {
    const testInstace = wrapper.root;
    expect(testInstace.props).toEqual({});
  });
});

describe("GridView component", () => {
  let wrapper = renderer.create(<GridView username="seyi" />);

  it("renders succesfully with props", () => {
    expect(wrapper.toJSON()).toMatchSnapshot();
  });

  it("renders with props", () => {
    const testInstace = wrapper.root;
    expect(testInstace.props).toEqual({ username: "seyi" });
  });
});
