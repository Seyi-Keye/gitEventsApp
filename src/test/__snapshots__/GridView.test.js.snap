exports[`GridView component renders succesfully 1`] = `
<div
  style={
    Object {
      "margin": "10px",
    }
  }>
  <div>
    
    <p>
       No results 
    </p>
  </div>
</div>
`;

exports[`GridView component renders succesfully with props 1`] = `
<div
  style={
    Object {
      "margin": "10px",
    }
  }>
  <div>
    
    <p>
       No results 
    </p>
  </div>
</div>
`;
